src/pages/graphql/schema/catalog-service/queries/categories.md

```diff
@@ -0,0 +1,195 @@
+---
+title: categories query | GraphQL Developer Guide
+edition: ee
+description: Describes how to construct and use the Catalog Service categories query.
+---
+
+# categories query
+
+The `categories` query returns categories and the products assigned to those categories.
```

Comment:
There's nothing in the schema that indicates the ability to show which products are assigned. If the query can do that, you need another example.